frontend/src/Components/CustomizedInputBase.js

```diff
@@ -25,20 +27,29 @@ export function CustomizedInputBase(props) {
   const [input, setInput] = React.useState('');
   const [subgraph, setSubgraph] = React.useState('');
   const Search = async (event) => {
+    let res;
     event.preventDefault();
     // console.log(input)
     props.apiCalled(true)
-
-    // const res = await axios.post('http://127.0.0.1:5000/api/v1/dashboard', {
-    //   input: input,
-    //   subgraph: subgraph,
-    // })
+    
+    try{
+      res = await axios.post(`${backend_url}/api/v1/dashboard`, {
```

Comment:
I am cherrypicking this

Reply:
Let me push a new commit(commenting the viz part)